frontend/src/scenes/toolbar-launch/AuthorizedUrls.scss

```diff
@@ -0,0 +1,6 @@
+.break-word {
+    overflow-wrap: break-word;
+    width: 50%;
+    text-overflow: ellipsis;
+    white-space: normal !important;
+}
```

Comment:
@gitstart For future reference, we should never be adding classes like this in a component specific CSS file as they would then exist globally and could break things.

If we want to add utility classes they should be done here (and very sparingly) https://github.com/PostHog/posthog/blob/master/frontend/src/styles/utilities.scss

All other SCSS files should follow BEM style